### ISSUE TITLE
lametric binding not working anymore with open hab 2.2.0-SNAPSHOT

### ISSUE THREAD

**Issue by reyem** [syphr42/openhab2-addons#2]
just did an apt-get update/upgrade to install to the newest snapshot. However, lametric binding fails to start because of a missing dependency.

in kara the diagnosis for the binding says
```
bundle:diag 197
LaMetric Time Binding (197)
---------------------------
Status: Installed
Unsatisfied Requirements:
osgi.wiring.package; filter:="(osgi.wiring.package=com.eclipsesource.jaxrs.provider.gson)"
```

So I installed it manually with 

`bundle:install http://central.maven.org/maven2/com/eclipsesource/jaxrs/provider-gson/2.3/provider-gson-2.3.jar`

that made it working again. However, we should make it work out of the box

**Comment by syphr42** [comment]
You are correct. The issue is the same as the one I fixed in openhab/openhab2-addons#2388 and it is due to eclipse/smarthome#3682.

All we need to do is import the provider-gson library. I'll get that done soon.

**Comment by syphr42** [comment]
This should be fixed with 67da60e8765bf4e59e40078f8c70ff50689c0543.
